code/game/gamemodes/nuclear/nuclearbomb.dm

```diff
@@ -515,16 +515,12 @@ This is here to make the tiles around the station mininuke change when it's arme
 	user.visible_message("<span class='suicide'>[user] is going delta! It looks like [user.p_theyre()] trying to commit suicide!</span>")
 	playsound(user.loc, 'sound/machines/Alarm.ogg', 50, -1, 1)
 	var/end_time = world.time + 100
-	var/orig_color = user.color
 	while(world.time < end_time)
 		if(!user)
 			return
-		if(user.color == "#FF0000")
-			user.color = "#00FF00"
-		else
-			user.color = "#FF0000"
+		user.add_atom_colour(RANDOM_COLOUR, ADMIN_COLOUR_PRIORITY)
```

Comment:
This should be flickering between red and green, which appears to have been accidentally reverted?